### DISCUSSION TITLE
Sleeping

### DISCUSSION THREAD

**Discussion by VintageGeek** [ORelio/Minecraft-Console-Client#1843] [Q&A]
Hi all

Thanks for the great client!  I was wondering how to sleep with the client when others sleep on the server.  Is there a way to detect sleeping players?  Also, how would the client use a bed to sleep.  Would that be the UseBlock command?

Thanks

**Comment by ReinforceZwei** [comment]
MCC didn't implement sleeping on the bed nor detecting sleeping players.

**Reply by VintageGeek** [reply]
Ok, thanks.  I think I'll just have the client reconnect with a bit of a delay when it detects the (#/#) players sleeping text
